pkgs/development/lua-modules/overrides.nix

```diff
@@ -332,15 +332,23 @@ with prev;
   # we shouldn't use luarocks machinery to build complex cmake components
   libluv = pkgs.stdenv.mkDerivation {
 
-    inherit (prev.luv) pname version meta src;
+    pname = "libluv";
+    inherit (prev.luv) version meta src;
 
       cmakeFlags = [
         "-DBUILD_SHARED_LIBS=ON"
         "-DBUILD_MODULE=OFF"
-        "-DWITH_SHARED_LIBUV=ON"
+        "-DWITH_SHARED_LIBUV=TOTO"
```

Comment:
is this a typo?

Reply:
good catch. I was testing at some point because libluv wasn't taking into account my changes. Thanks for the review, much appreciated.